src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -1,15 +1,35 @@
 package core.mate.academy.model;
 
-/**
- * Add some custom fields that could be only in Bulldozer
- * Do not remove no-args constructor
- */
 public class Bulldozer extends Machine {
+    private int enginePower;
+    private String dumpType;
+
     public Bulldozer() {
     }
 
+    public Bulldozer(int enginePower, String dumpType) {
+        this.enginePower = enginePower;
+        this.dumpType = dumpType;
+    }
```

Comment:
Since this class `extends Machine` you should take advantage of it and use `Machine` constructor here with all fields by simply calling `super(arguments here)`;. Fir example like this:

```java
public Bulldozer(String name, String color, Double bladeWidth, Integer cabCapacity) {
        super(name, color);
        this.bladeWidth = bladeWidth;
        this.cabCapacity = cabCapacity;
    }
```